### ISSUE TITLE
add few features

### ISSUE THREAD

**Issue by karokarok** [sharpbrowser/SharpBrowser#23]
Hello.
i would like you request for a few standard features that maybe still missing in this project.

1.History Browser
2.Home button
3.Status Bar in bottom
4.Multiple thread for Download manager
5.Pages Zoom
6.Pages Snapshot/ScreenShot
7.Password Manager
8.Auto Update
9.Private Tab
10.Tab Task Manager
11.AdsBlock Build-in Extension/Plugins just like opera
12.Bookmarks Manager
13.Automatic Bug Reports on Exceptions Error.

**Comment by robinrodricks** [comment]
Of course, we'll begin work on it shortly after the payment is made :)

**Comment by Daszbin** [comment]
I made a better version of SharpBrowser called Coder Nut it’s free to download its has all the features you want
Dark Mode
Bookmark Manager
History
Error reporting
Open pdf, doc & docx, excel files, PowerPoint files, music files and etc.
Ever changing updates like on a daily basis 
Check it out
https://gamerslube.weebly.com/downloads.html
Download Coder Nut

**Comment by Daszbin** [comment]
I made a better version of SharpBrowser called Coder Nut it’s free to download its has all the features you want
Dark Mode
Bookmark Manager
History
Error reporting
Open pdf, doc & docx, excel files, PowerPoint files, music files and etc.
Ever changing updates like on a daily basis
Check it out
https://gamerslube.weebly.com/downloads.html
Download Coder Nut

**Comment by robinrodricks** [comment]
Too large scope. As this is a FOSS project anyone who has the time and energy to rebuild firefox can take it up themselves.
